products/vault_selinux/vault.te

```diff
@@ -59,8 +61,11 @@ allow vault_t self:fifo_file rw_fifo_file_perms;
 allow vault_t self:unix_stream_socket create_stream_socket_perms;
 # and network sockets too
 allow vault_t self:netlink_route_socket { bind create getattr nlmsg_read };
-allow vault_t self:unix_dgram_socket { bind create getattr setopt };
+allow vault_t self:unix_dgram_socket { bind create getattr setopt connect sendto };
 allow vault_t self:tcp_socket { accept bind create getattr listen setopt read write };
+allow vault_t unreserved_port_t:tcp_socket name_connect;
```

Comment:
@eric-eisenhart, do you know what denials caused these additions? My understanding is you could trigger them with something like an external KMS or potentially a database credentials secrets backend? [`unreserved_port_t`](https://wiki.gentoo.org/wiki/SELinux/Tutorials/Managing_network_port_labels) is a broad range of ports, so my one hesitation is perhaps there's a small-ish set of more descriptive labels here that we could document better why we want these specific ports (databases, &c). But then again perhaps not worth it and perhaps better to add a comment describing why this was suggested :-)